OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameChatLogic.cs

```diff
@@ -47,7 +47,7 @@ public IngameChatLogic(Widget widget, OrderManager orderManager, World world, Mo
 			chatTraits = world.WorldActor.TraitsImplementing<INotifyChat>().ToArray();
 
 			var players = world.Players.Where(p => p != world.LocalPlayer && !p.NonCombatant && !p.IsBot);
-			disableTeamChat = world.IsReplay || world.LobbyInfo.IsSinglePlayer || (world.LocalPlayer != null && !players.Any(p => p.IsAlliedWith(world.LocalPlayer)));
+			disableTeamChat = world.IsReplay || world.LobbyInfo.IsSingleHuman || (world.LocalPlayer != null && !players.Any(p => p.IsAlliedWith(world.LocalPlayer)));
```

Comment:
Actually I think we can drop the check here completely.  The filtered `players` check already makes sure that you have two humans on the same team.